plasmapy/formulary/tests/test_densities.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for functionality contained in `plasmapy.formulary.frequencies`."""
+import astropy.units as u
+import numpy as np
+import pytest
+
+from plasmapy.formulary.densities import plasma_critical_density
+from plasmapy.formulary.frequencies import plasma_frequency
+
+n_i = 5e19 * u.m**-3
```

Comment:
I would move these definitions into the test, since (at least for now) they are not being used by multiple tests